syntax = "proto3";

message Thrusters {
    int32 thruster_1 = 1;
    int32 thruster_2 = 2;
    int32 thruster_3 = 3;
    int32 thruster_4 = 4;
    int32 thruster_5 = 5;
    int32 thruster_6 = 6;
    int32 thruster_7 = 7;
    int32 thruster_8 = 8;

}
